.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,15 @@
+## What?
+
+Describe what changes you are making. i.e. `Adding additional eslint configs to catch potential bugs.`
```

Comment:
🍹 im gonna be that guy and point out its probably better to use `e.g.` here not `i.e.`

Reply:
Love that better! Thanks for looking for that!